src/Kmd.Logic.CitizenDocuments.Client/CitizenDocumentsClient.cs

```diff
@@ -44,22 +46,29 @@ public CitizenDocumentsClient(HttpClient httpClient, LogicTokenProviderFactory t
         /// <summary>
         /// Uploads the single citizen document.
         /// </summary>
+        /// <param name="configurationId">configuration id.</param>
+        /// <param name="retentionPeriodInDays">retention period.</param>
+        /// <param name="cpr">cpr.</param>
+        /// <param name="documentType">document type.</param>
+        /// <param name="document">document.</param>
+        /// <param name="documentName">document name.</param>
```

Comment:
Why bother repeating the parameter names in the documentation? Wouldn't it be better to either:

1. Remove the documentation entirely
2. Make it actually meaningful and useful

Reply:
Made param name description meaningful.